packages/typechain-target-fuels/src/parser/abiParser.ts

```diff
@@ -74,7 +76,7 @@ export interface DocumentationResult {
  */
 function parseFunctionDeclaration(
   abiPiece: RawAbiDefinition,
-  registerStruct: (struct: TupleType) => void,
+  registerStruct: (datum: StructType | EnumType | TupleType) => void,
```

Comment:
I see that this combination repeats a lot maybe create a single type for it;
`type DatumType = StructType | EnumType | TupleType`?